scripts/dts/edtlib.py

```diff
@@ -1229,7 +1235,7 @@ def _check_binding(binding, binding_path):
     if "properties" not in binding:
         return
 
-    ok_prop_keys = {"description", "type", "category", "constraint", "enum"}
+    ok_prop_keys = {"description", "type", "category", "constraint", "enum", "const"}
```

Comment:
Could put `"const"` on a new line, just for consistency:

```python
ok_prop_keys = {"description", "type", "category", "constraint", "enum",
                "const"}
```

Reply:
Fixed.